rm-datasource/src/main/java/io/seata/rm/datasource/exec/BaseTransactionalExecutor.java

```diff
@@ -459,6 +469,39 @@ protected TableRecords buildTableRecords(Map<String, List<Object>> pkValuesMap)
         }
     }
 
+    protected void doIgnoreCheckColumns(Set<String> columnNames, StringJoiner selectSQLJoin) {
+
+        Set<String> columns = new TreeSet<>(String.CASE_INSENSITIVE_ORDER);
+        columns.addAll(columnNames);
+        Map<String, Set<String>> ignoreCheckFields = new HashMap<>();
+
+        try {
+            if (IGNORE_NOCHECK_COLUMNS.length() > 0) {
+                Map<String, String> maps = (Map) JSON.parse(IGNORE_NOCHECK_COLUMNS);
```

Comment:
use jackson, may be in init or listener time parse better

Reply:
get i'm try optimize